vector/src/main/AndroidManifest.xml

```diff
@@ -186,9 +192,9 @@
         <activity android:name=".features.webview.VectorWebViewActivity" />
 
         <!-- Activity to intercept links coming from a web instance -->
+        <!-- exported="true" is required to handle android.intent.action.VIEW for URL redirection-->
         <activity
             android:name=".features.link.LinkHandlerActivity"
-            android:enabled="true"
```

Comment:
Can we keep this line please? I know `true` is the default value, but it's here to help fork maintainers that it can be disabled here.